pyproject.toml

```diff
@@ -1,9 +1,33 @@
 [build-system]
-requires = [
-    "setuptools>=42",
-    "wheel"
-]
+requires = ["setuptools>=61.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+authors = [
+  { name="Jörg F. Unger", email="joerg.unger@bam.de" },
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+]
+dependencies = [
+  "fenics-dolfinx",
+  "pint",
+]
+description = "Implementation of structural problems for concrete structures using FEniCSx"
+license = {file = "LICENSE"}
+name = "fenicsxconcrete"
+readme = "REAMDE.md"
+requires-python = ">=3.7"
+version = "0.0.1"
```

Comment:
can we somehow move that to either the git tag or a unique location at least?

Reply:
thanks, I added a ToDo. I was just simplyfying the setup to get a first working version which can then be improved step-by-step. This is why I set this PR to draft